### DISCUSSION TITLE
Category Not Saved?

### DISCUSSION THREAD

**Discussion by bosborn** [community/community#3013] [Discussions] [answered]
The selected category seems to always default back to General when saved.  Not sure if I'm missing something.
This happened in Chrome and Safari.  On Firefox I couldn't even select a category.

https://github.com/bosborn/test/discussions

**Comment by becca** [comment, marked as answer]
This was a bug that I believe should be fixed now 🤞

**Reply by bosborn** [reply]
It is working now, thanks!
